config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
+  get 'users/show'
   devise_for :users
 
   resources :dishes
   resources :menu_days, only: %i[index show]
+  resources :orders, only: %i[index new create]
+  resources :users, only: [:show, :edit]
```

Comment:
Edit can be only for admins or edit only your own page. Even if I filled another id in the url